src/data-loader.ts

```
@@ -794,6 +795,55 @@
+			return Result.unwrap(fetcher.calculateCostFromTokens(data.message.usage, data.message.model), 0);
+		}
+
+		return 0;
+	}
+
+	if (mode === 'max') {
+		// Max mode: MAX(saved statusline costs, pre-calculated costUSD, token-based calculation)
+		const costs: number[] = [];
+
+		// Get saved statusline cost
+		if (data.sessionId != null) {
```

ESLint

[error] Unsafe member access .sessionId on an `error` typed value.

(ts/no-unsafe-member-access)